Raivo/Forms/MiscellaneousForm.swift

```diff
@@ -287,7 +287,7 @@ class MiscellaneousForm {
         form +++ Section("Data", { section in
             section.tag = "data"
             section.hidden = Condition(booleanLiteral: !authenticated)
-            section.footer = HeaderFooterView(title: "Your data will be exported in an encrypted ZIP archive (using your encryption password).")
+            //section.footer = HeaderFooterView(title: "Your data will be exported in an encrypted ZIP archive (using your encryption password).")
```

Comment:
Why is this commented?

Reply:
That was me experimenting with how that section looks with/without a footer. Each option's title and respective page are self explanatory, but I couldn't make up my mind on whether to delete or rephrase the footer, so I commented it out for now.